From ccba4b30dc7ceb1ad951edfe688f0fc5989b6b36 Mon Sep 17 00:00:00 2001
From: Aurelien Jarno <aurelien@aurel32.net>
Date: Sun, 19 Apr 2009 15:31:18 +0200
Subject: [PATCH 1/2] lib: do not include <linux/fd.h> in ismounted.c

<linux/fd.h> is included for no reason in lib/ismounted.c. This
obviously breaks on non-Linux systems.

Signed-off-by: Aurelien Jarno <aurelien@aurel32.net>
---
 lib/ismounted.c |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/lib/ismounted.c b/lib/ismounted.c
index 0481c77..28ae325 100644
--- a/lib/ismounted.c
+++ b/lib/ismounted.c
@@ -11,7 +11,6 @@
 #include <stdlib.h>
 #include <errno.h>
 #include <fcntl.h>
-#include <linux/fd.h>
 #include <mntent.h>
 #include <string.h>
 #include <sys/stat.h>
-- 
1.6.2.3

